realm/realm-library/src/androidTest/java/io/realm/RealmJsonTests.java

```diff
@@ -1178,4 +1183,328 @@ public void createObjectFromJson_objectWithPrimaryKeySetValueDirectlyFromStream(
         assertEquals(1, owners.get(1).getId());
         assertEquals("bar", owners.get(1).getName());
     }
+
+    private void createNullablePrimaryKeyObjectFromJson(Realm testRealm, Class <? extends RealmObject> clazz, String jsonString) throws JSONException {
```

Comment:
These should be `createObjectFromJson_nullablePrimaryKey`, same for the others below: `<method_test>_<description>`